server/lib/util.js

```diff
@@ -12,6 +12,29 @@ module.exports.queryCondition = queryCondition;
 module.exports.take = take;
 module.exports.isTrueString = isTrueString;
 module.exports.isFalsy = isFalsy;
+module.exports.sqlDate = sqlDate;
+
+/**
+ * @function sqlDate
+ * @param {date} date A date
+ * @return {string}
+ * @description convert a date in 'yyyy-mm-dd' format
+ */
+function sqlDate(date) {
```

Comment:
Why are you using this function?

Better version: `moment(date).format('YYYY-mm-dd');`